### PR TITLE
Закоммитил

### PR DESCRIPTION
**Pull request by Eugenneer**
фыв

### CODE REVIEW

File: src/screens/Dota2/index.tsx [hunk 1]
```diff
@@ -50,22 +60,65 @@ class Dota extends React.Component<Props, State> {
...
 
     onPress = (element: DotaHero) => {
-        if (this.state.raiantTeam.length < PLAYERS_IN_TEAM){
-            let radiant = this.state.raiantTeam;
-            radiant.push(element.id); 
+        if (this.state.pickRediant){ 
+            if (this.state.radiantTeam.length < PLAYERS_IN_TEAM){
+                let newDataSorce = this.state.dataSource
+                    element.index = newDataSorce.indexOf(element)
+                    newDataSorce.splice(element.index,1);
```
Comment (kotnor) [inline src/screens/Dota2/index.tsx]: Space between ',' and 1. Remove ;

File: src/screens/Dota2/index.tsx [hunk 2]
```diff
@@ -50,22 +60,65 @@ class Dota extends React.Component<Props, State> {
...
+                    element.index = newDataSorce.indexOf(element)
+                    newDataSorce.splice(element.index,1); 
+                let radiant = this.state.radiantTeam
+                    radiant.push(element); 
+                this.setState({
+                    radiantTeam: radiant,
+                    dataSource: newDataSorce
+                });
+            }
+        }else if (this.state.pickDier){
```
Comment (kotnor) [inline src/screens/Dota2/index.tsx]: Add a space between } and else

File: src/screens/Dota2/index.tsx [hunk 3]
```diff
@@ -76,37 +129,81 @@ class Dota extends React.Component<Props, State> {
...
+                        </View>
+                    </TouchableHighlight>    
                 </View>
                 <View style={{flexDirection: 'row'}}>
-                    <View style={[styles.icon, styles.secondTeamIcon]}><Text>Hello</Text></View>
-                    <View style={[styles.icon, styles.secondTeamIcon]} />
-                    <View style={[styles.icon, styles.secondTeamIcon]} />
-                    <View style={[styles.icon, styles.secondTeamIcon]} />
-                    <View style={[styles.icon, styles.secondTeamIcon]} />
+                <TouchableHighlight onPress = {() => this.onRemove(this.state.dierTeam[0])} underlayColor="white">
```
Comment (kotnor) [inline src/screens/Dota2/index.tsx]: Add tabulation

File: src/screens/Dota2/index.tsx [hunk 4]
```diff
@@ -11,26 +11,36 @@ const options = {date_max: '1529366400', matches_requested : 2}; //start_at_matc
...
 
 }
 
 interface State {
     isLoading: boolean
+    pickRediant: boolean
+    pickDier: boolean
     dataSource: Array<any>
-    raiantTeam: Array<any>
+    radiantTeam: Array<any>
```
[the remark this answers is not in the document]
Comment (Eugenneer) [reply src/screens/Dota2/index.tsx]: Can I just use the already existing DotaHero type? This type {DotaHero} has all the necessary attributes.

**Comment by Eugenneer** [comment]
Я все сделал

File: src/screens/Dota2/index.tsx (hunk 4 above)
[the remark this answers is not in the document]
Comment (kotnor) [reply src/screens/Dota2/index.tsx]: @Eugenneer Yes, you should

**Pull request merged**